<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-MHRN7JA" ignored="0" name="Default test" tests="2" failures="0" timestamp="2020-11-02T12:39:05 IST" time="0.069" errors="0">
  <testcase name="@BeforeTest initializeTest" time="0.01" classname="concepts.framework.practice.ValidateTitle">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at concepts.framework.practice.BaseClass.initializeDriver(BaseClass.java:60)
at concepts.framework.practice.ValidateTitle.initializeTest(ValidateTitle.java:17)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.beforeRun(TestRunner.java:607)
at org.testng.TestRunner.run(TestRunner.java:578)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest initializeTest -->
  <testcase name="@AfterTest cleanUp" time="-1.60430094594E9" classname="concepts.framework.practice.ValidateTitle">
    <skipped/>
  </testcase> <!-- @AfterTest cleanUp -->
  <testcase name="headerTextValidation" time="0.0" classname="concepts.framework.practice.ValidateTitle">
    <skipped/>
  </testcase> <!-- headerTextValidation -->
  <testcase name="titleTextValidation" time="0.0" classname="concepts.framework.practice.ValidateTitle">
    <skipped/>
  </testcase> <!-- titleTextValidation -->
</testsuite> <!-- Default test -->
